src/app/shared/functions/players.ts

```diff
@@ -16,7 +17,46 @@ export const filterPlayers = (params: FilterPlayersParams) => {
 
       return playerPositions.some(({ position }) => position.key === selectedPosition)
     })
+    return filteredPlayerList
   }
 
+  // If No Selected Position (i.e. Bench 19-23)
+  // Filter players by selectedTeamSheetLayoutId below:
+  filteredPlayerList = players.filter(player => {
+    const { playerPositions } = player
+
+    let filterPositions: String[] = []
+
+    switch (selectedTeamSheetLayoutId) {
+      case ('19' || '20'):
```

Comment:
i think we should write this logic a little differently, we already have a file where we set which positions are available on which teamSheetLayoutIds `src/app/shared/functions/teamSheet.ts`

if we are now allowing multiple positions on some teamSheetLayoutIds i think we need to re-write the layoutData like the following, and change the logic to handle an array of positions instead of a string
```
{ teamSheetLayoutId: '1', positions: ['prop'], positionTitle: 'Prop', className: 'col-start-0 col-span-2' },
{ teamSheetLayoutId: '19', positions: ['lock', 'backrow'] },
```

Reply:
Yeh I had initially thought of doing this and it is a better way. Will update.